features/bootstrap/JoindInApiContext.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+use GuzzleHttp\Client;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Webmozart\Assert\Assert;
+
+class JoindInApiContext extends BaseContext
+{
+    use FixturesTrait;
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /** @var string|null */
+    private $raffleId;
+    /** @var array|null */
+    private $picked;
```

Comment:
Same here as previous^^

Reply:
Same answer, we need to store the information between steps